utils/generatePairs.ts

```diff
@@ -2,14 +2,13 @@ import { App } from "@slack/bolt";
 import { UsersDataManager } from "../database/usersDataManager";
 import { PoolDataManager } from "../database/poolDataManager";
 
-//TODO: change this to actual channel id when deploying
-const TESTING_CHANNEL_ID = "C02J6R0SUSX";
+const CHANNEL_ID = "C04GHB54U30";
```

Comment:
i've decided to create a permanent slack channel for the slack bot called `#boba-buddies`
https://sandboxneu.slack.com/archives/C04GHB54U30

Reply:
this will be where everyone wants to participate should be in